### PR TITLE
Updating mutation createFolder

### PR DESCRIPTION
**Pull request by lionelakpagni**
Errors are below:
"eng-GB" => eng_GB
languageCode => language

| Question      | Answer
| ------------- | ---
| JIRA Ticket   | 
| Versions      |

### CODE REVIEW

**Comment by DominikaK** [comment]
@lionelakpagni, when using your proposed change I'm getting the following error (with GraphiQL):
`Syntax Error: Invalid number, expected digit but got: \"G\"`

File: docs/api/graphql_operations.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Create a new Folder as a child of Location `2` with:
 ```
 mutation createFolder {
   createFolder(
-    languageCode: "eng-GB"
+    language: eng-GB
```
Comment (adriendupuis) [inline docs/api/graphql_operations.md:16]:
```suggestion
    language: eng_GB
```

**Comment by lionelakpagni** [comment]
> @lionelakpagni Could you also update the `mutation updateFolder` at line 46 with `language: eng_GB` please? (no quote, underscore instead of minus)

Done. I missed this part, however I mentioned it in the conversation

**Comment by lionelakpagni** [comment]
Hey all, I close this PR.
Please, refer to this: https://github.com/ezsystems/developer-documentation/pull/1052/f

**Pull request closed without merging**